src/UI/button/Button.jsx

```diff
@@ -0,0 +1,10 @@
+import styles from './Button.scss';
+
+function Button(props) {
```

Comment:
you can destructure props like
```suggestion
function Button({ text, ...props}) {
```

that way it's clear what the component receives and prevents some console errors